backstory/src/main/java/com/google/sps/perspective/ContentDecider.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.perspective;
+
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeType;
+import java.util.Map;
+
+/**
+ * Makes a decision on whether or not the story (or content) is appropriate
+ * using analyses from Perspective API.
+ */
+public class ContentDecider {
```

Comment:
ContentDecider is a bit weird for a class that can't be an object. A rename to ContentDecisionTools may be better.

Reply:
As Joe noted it has controller logic, and is not just a value container, which is why I named it that. However, it might not be a good name for a static class, which it wasn't clear it would be at the outset. @ykjchen Do you think there should be a name change?

Reply:
Ah, I didn't realize this wasn't to be used as an instance.

If just a collection of static functions, ContentDecisionTools works for me, or if you want to use other Java classes as a guide, perhaps the name could model after Collections:

```
public class ContentDecisions {};
```

Then you'd do:

```
ContentDecisions.makeDecision(...);
```

I don't have a strong preference here.

Reply:
I think ```ContentDecisions``` would work well. I'll change that.

Reply:
Done